### PR TITLE
Add interlinks and minor content changes

### PR DESCRIPTION
**Pull request by rodmiromind**

### CODE REVIEW

File: content/en/docs/faq/observability/byop.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Bring Your Own Prometheus"
+title: "Install Your Own Prometheus"
```
Comment (Sherlock113) [inline content/en/docs/faq/observability/byop.md:2]: Just my two cents:

I would like to confirm here. In fact `Bring Your Own (BYO)` is often used in IT, such as Bring Your Own Software. You can also find similar expressions in [Google Doc](https://cloud.google.com/vpc/docs/using-bring-your-own-ip). The title is to tell readers that they can integrate external Prometheus into KubeSphere. So can we continue to use `Bring Your Own Prometheus`?

Comment (rodmiromind) [reply content/en/docs/faq/observability/byop.md:2]: Ok

**Review by FeynmanZhou — APPROVED** [review]

**Pull request merged**